ecommerce/models.py

```diff
@@ -46,6 +50,43 @@ def __str__(self):
         """Description for Order"""
         return "Order {}, status={} for user={}".format(self.id, self.status, self.user)
 
+    def to_dict(self):
+        """
+        Get a serialized representation of the Order and any attached Lines
+        """
+        data = serialize_model(self)
+        data['lines'] = [serialize_model(line) for line in self.line_set.all()]
+        return data
```

Comment:
i'm sort of confused by this pattern now. why aren't we just defining a new django rest framework ModelSerializer to perform this serialization? especially looking at `'lines'` - that is explicitly looping through and serializing a related object, which is one of the things that DRF serializers takes care of for you. also, i don't think it's optimal that this class gets its serialization by calling `obj.to_dict()` while the `FinancialAid` class does the same by invoking `serialize_model(obj)`. i think we should have a mixin/decorator for any class that needs a serialized version of its objects for an audit table. that way we could have a consistent API for auditing. 

we should probably have a base model for `*Audit` tables too. `acting_user`, `data_before/after` and `save_and_log` should behave pretty much the same for all audit tables, right?